.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,26 @@
+name: Docker image build
+
+on: [push, release]
```

Comment:
Both on push and on release? This looks redundant, as our releases are nothing but tags attached to commits that have been pushed. Maybe it would make sense to build containers on releases, if they are tagged with the release number.

Reply:
Yes, that makes sense.

I just added both so that if a released is triggered manually, and there is some version string utilization in the build, it will reflect in the container.

Reply:
Then let's just replace `release` with `repository_dispatch`.